packages/vue-app/template/mixins/fetch.client.js

```diff
@@ -72,7 +72,7 @@ function createdFullStatic() {
     this._fetchKey = this.$options.fetchKey.call(this, getCounter)
   } else {
     const key = 'string' === typeof this.$options.fetchKey ? this.$options.fetchKey : defaultKey
-    this._fetchKey = key + ':' + getCounter(key)
+    this._fetchKey = key + getCounter(key)
```

Comment:
Wouldn't better if adding `:` to server generated keys? key=[number] can potentially allow _fetchKey conflicts